old_optimizers/__init__.py

```diff
@@ -0,0 +1,71 @@
+from thumbor.config import Config
```

Comment:
Why did you move all the optimizers to old_optimizers?

Reply:
I still need to move one by one to the new format.